.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit
+    rev: v1.20.0
```

Comment:
```suggestion
    rev: v2.1.1
```

Reply:
Good point, I haven't kept track of pre-commit versions having advanced. I'll fold that into my next push which should hopefully also fix the Travis CI issues in shellcheck HEAD.